controllers/factory/vmagent.go

```diff
@@ -142,13 +151,23 @@ func CreateOrUpdateVMAgent(ctx context.Context, cr *victoriametricsv1beta1.VMAge
 			}
 		}
 	} else {
+		// To save compatibility in the single-shard version still need to fill in placeholders
+		placeholders := map[string]string{"shard-num": "0"}
```

Comment:
Lets create global var

 vmagentShardPlaceHolder = `%SHARD_NUM%` and refer it at docs.

 E.g. placeholders := map[string]string{ vmagentShardPlaceHolder: "0"}

Reply:
done